src/views/routes.js

```diff
@@ -1,14 +1,15 @@
 import React from 'react';
 import { Switch, Route } from 'react-router-dom';
-import NotFound from '../components/sharedComponents/notFound';
+// import sample from '../components/languages/sample';
+import sample from '../components/sharedComponents/notFound';
 import Home from '../components/sharedComponents/home';
 import LoginPage from '../components/authentication/Login';
 
 const Routes = () => (
   <Switch>
     <Route exact path="/" component={Home} />
     <Route exact path="/login" component={LoginPage} />
-    <Route component={NotFound} />
+    <Route component={sample} />
```

Comment:
isn't the name NotFound more understandable than Sample on the not found component